ImageGallery/src/org/sleuthkit/autopsy/imagegallery/ImageGalleryController.java

```diff
@@ -200,11 +202,6 @@ void setStale(Boolean b) {
         Platform.runLater(() -> {
             stale.set(b);
         });
-        try {
-            new PerCaseProperties(Case.getCurrentCaseThrows()).setConfigSetting(ImageGalleryModule.getModuleName(), PerCaseProperties.STALE, b.toString());
-        } catch (NoCurrentCaseException ex) {
-            Logger.getLogger(ImageGalleryController.class.getName()).log(Level.WARNING, "Exception while getting open case."); //NON-NLS
```

Comment:
Am I correct that we are thowing out the idea of staleness for the whole database and instead keeping track per datasource.  That means the only property left in PerCaseProperties is 'enabled'  do we want a whole class just for this?  Is there somewhere else it could go?

Reply:
Well we got rid of tracking the DB staleness as a whole via the property.  The DB is stale is any of the data sources is stale (or there are new data sources not synced)